pkg/webhook/admission/servingruntime/servingruntime_webhook.go

```diff
@@ -223,39 +220,31 @@ func validateMultiNodeSpec(newSpec *v1alpha1.ServingRuntimeSpec, existingSpec *v
 					return errors.New(DisallowedWorkerSpecTensorParallelSizeEnvError)
 				}
 
-				if isUnknownGPUType, err := utils.IsUnknownGpuResourceType(container.Resources, newSpec.Annotations); err != nil {
+				if hadUnknownGpuType, err := utils.HasUnknownGpuResourceType(container.Resources, newSpec.Annotations); err != nil {
 					return err
-				} else if isUnknownGPUType {
+				} else if hadUnknownGpuType {
 					return fmt.Errorf(InvalidUnknownGPUTypeError, constants.InferenceServiceContainerName)
 				}
 			}
 		}
 		workerContainer := newSpec.WorkerSpec.Containers[0]
 		if workerContainer.Name == constants.WorkerContainerName {
-			if isUnknownGPUType, err := utils.IsUnknownGpuResourceType(workerContainer.Resources, newSpec.Annotations); err != nil {
+			if hadUnknownGpuType, err := utils.HasUnknownGpuResourceType(workerContainer.Resources, newSpec.Annotations); err != nil {
 				return err
-			} else if isUnknownGPUType {
+			} else if hadUnknownGpuType {
 				return fmt.Errorf(InvalidUnknownGPUTypeError, constants.WorkerContainerName)
 			}
 		}
 
-		if newSpec.WorkerSpec.PipelineParallelSize == nil {
-			return errors.New(MissingPipelineParallelSizeValueError)
-		}
-
-		if newSpec.WorkerSpec.TensorParallelSize == nil {
-			return errors.New(MissingTensorParallelSizeValueError)
-		}
-
 		// WorkerSpec.PipelineParallelSize should not be less than 2.
 		pipelineParallelSize := *newSpec.WorkerSpec.PipelineParallelSize
-		if pipelineParallelSize < 2 {
+		if pipelineParallelSize < constants.DefaultPipelineParallelSize {
```

Comment:
We don't have a defaulting webhook, right?
If not, I think we, at least, need the `nil` checks. Otherwise, the de-reference of `*newSpec.WorkerSpec.PipelineParallelSize` will generate a panic.

This also applies to the following lines.

Reply:
nil check is already there at the first line
```
	if existingSpec.WorkerSpec != nil && newSpec.WorkerSpec == nil {
		return errors.New(DisallowedRemovingWorkerSpecFromServingRuntimeError)
	}

	if newSpec.WorkerSpec != nil {
```

Reply:
right. I only checked workerSpec. I updated